test/from-draft7/to-2019-09/002-unknown-maxContains.json

```diff
@@ -0,0 +1,24 @@
+[
+    {
+        "title": "maxContains present in the schema",
+        "from": {
+            "maxContains": "newbie"
```

Comment:
`maxContains` is also present in 2019-09: https://www.learnjsonschema.com/2019-09/validation/maxcontains/.